packages/video_player/lib/video_player.dart

```diff
@@ -149,7 +149,8 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
   /// The name of the asset is given by the [dataSource] argument and must not be
   /// null. The [package] argument must be non-null when the asset comes from a
   /// package and null otherwise.
-  VideoPlayerController.asset(this.dataSource, {this.package})
+  VideoPlayerController.asset(this.dataSource,
```

Comment:
To make the public methods easy to maintain in the future. Do you mind make a new class "VideoPlayerOptions" and put the  `mixWithOthers` as one of the members.